WhoIsThatServer.Recognition/Recognition/RecognitionServices.cs

```diff
@@ -133,5 +133,28 @@ public async Task<bool> InsertPersonInToGroup(ImageModel imageModel)
 
             return true;
         }
+
+        /// <summary>
+        /// Detects features of face
+        /// </summary>
+        /// <param name="imageModel">Object which is sent from frontend</param>
+        /// <returns>Age, gender of face and whether glasses and facialhair are present</returns>
+        public async Task<Face[]> DetectFeaturesOfFace(ImageModel imageModel)
+        {
+            try
+            {
+                var faces = await _faceServiceClient.DetectAsync(RecUtil.GetStreamFromUri(imageModel.ImageContentUri), false, true, new FaceAttributeType[] {
+                    FaceAttributeType.Age,
+                    FaceAttributeType.Gender
+                });
+
+                return faces.ToArray();
```

Comment:
Are you sure `IEnumerable` is not enough?

Reply:
Yeah, probably it's enough, will remove that. Though it would throw exception if faces would not be found and I can catch it..

Reply:
I mean 'ToArray()' part throws exception